CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 otj-metrics
 ===========
 
+4.0.0
```

Comment:
@adamzr no because your changes are 3.0.0 (incompatible api change). This has the advantage of us being able to mentally say "Oh DropWizard 4.x == otj-metrics 4.x